DEVELOPER.md

```diff
@@ -28,6 +28,7 @@ There are three `npm run` commands that can help ensure consistent style:
 
 * `npm run format` will reformat almost everything _except_ `*.md`. This command uses Prettier, which does not have provide any customization options for Markdown files. Note that it _is_ running on `*.mdx` files.
 * `npm run lint` calls Markdownlint, the same tool used in the Visual Studio Code extension. It will only report style errors.
+* `npm run lint:folder --path={folder}` calls Markdownlint, the same tool used in the Visual Studio Code extension. It will only report style errors. eg: `npm run lint:folder --path=docs/reference/6-data-import`
```

Comment:
FYI, in the npm 11.x this generates a deprecation warning. And there are [many people](https://github.com/npm/cli/issues/8153) confused about what to do. Including me.

Let's accept this for now, because it is working fine. We may have to revisit it in the future.